cmd/kube-scheduler/app/server.go

```diff
@@ -168,6 +168,7 @@ func Run(cc schedulerserverconfig.CompletedConfig, stopCh <-chan struct{}, regis
 
 	// Create the scheduler.
 	sched, err := scheduler.New(cc.Client,
+		cc.InformerFactory,
```

Comment:
I think it is better to pass a different informerFactory. InformerFactory has a cache of informers internally, so this implementation expose informers that scheduler uses.
https://github.com/kubernetes/client-go/blob/8248d0a0e61a6cc77b53557918964cd1038d511f/informers/factory.go#L162-L183

Reply:
Sorry, what are you suggesting?

Reply:
I was worried that plugin implementers changes lister's cache by mistake and it causes unexpected behavior of scheduler core. But maybe I'm worried too much.

Reply:
since we are passing the sharedInformerFactory, lets remove all listers that are derived from it.

Reply:
Sure I’ll update the PR